### DISCUSSION TITLE
Decimal compare

### DISCUSSION THREAD

**Discussion by UsboKirishima** [community/community#141920] [Code Security] [answered]
### Select Topic Area

Show & Tell

### Body

Many people thinks that 0.1 + 0.2 = 0.3 but if you try to compare this in python 
```py
if 0.1 + 0.2 == 0.3: print('yes') else: print('no')```

**Comment by MuxN4** [comment, marked as answer]
@UsboKirishima I'd say its a consequence of how floating-point numbers are represented in binary
